ui/src/main/webapp/src/app/executions/execution-detail.component.html

```diff
@@ -47,6 +47,10 @@ <h3 i18n="Analysis status|Section">Status</h3>
                     <dt i18n="Analysis last modification">Last modification</dt>
                     <dd>{{execution.lastModified ? (execution.lastModified | date: 'short') : 'n/a'}}</dd>
                 </ng-container>
+                <ng-container *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports">
+                    <dt i18n="Analysis static report">Report</dt>
+                    <dd><a class="link" target="_blank" href="{{formatStaticReportUrl(execution)}}">Static Report <i class="fa fa-external-link"></i></a></dd>
```

Comment:
I don't think I'm a fan of just having "Report" only, I like when it gives you the action. I would either go with the first option, or a "View Report" link.